src/components/change-email/tests/change-email-controller.test.ts

```diff
@@ -52,5 +64,22 @@ describe("change email controller", () => {
         .calledOnce;
       expect(res.redirect).to.have.calledWith("/check-your-email");
     });
+
+    it("should render bad request", async () => {
+      const fakeService: ChangeEmailServiceInterface = {
+        sendCodeVerificationNotification: sandbox.fake.returns(true),
+      };
+
+      req.body.email = "test@test.com";
+      req.session.user = {
+        tokens: { accessToken: "token" },
+        email: "test@test.com",
+        state: { changeEmail: getInitialState() },
+      };
+      req.cookies.lng = "en";
+
+      await changeEmailPost(fakeService)(req as Request, res as Response);
+      expect(res.status).to.have.calledWith(HTTP_STATUS_CODES.BAD_REQUEST);
```

Comment:
this is a bad request because the new email is the same as the existing email?

Reply:
Yes from what I can see in the controller code